apps/src/code-studio/components/aiTutor/ai-tutor.module.scss

```diff
@@ -85,4 +85,4 @@ $default-spacing: 2px;
 
 .textArea {
     width: 80%;
-}
\ No newline at end of file
+}
```

Comment:
This file needs to come along with the PR since there aren't any meaningful updates here (looks like maybe your linter auto-added a new line?). 

To resolve you could: 
1.) manually adjust the file so it matches the formatting on staging and re-save it so the diff no longer shows OR 
2.)  run `git checkout staging apps/src/code-studio/components/aiTutor/ai-tutor.module.scss` on your feature branch which will revert this file to the staging version 

and then add, commit and push the revision.

Reply:
I think I addressed this!